tests/models/backbones/test_onnx.ipynb

```diff
@@ -478,4 +478,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 2
-}
```

Comment:
It has `for backbone_name in BACKBONES.class_dict.keys():` which will not be work, because now it `for backbone_name in BACKBONES.entrypoints.keys():`

Reply:
Fixed